gradebookng/impl/src/main/java/org/sakaiproject/grading/impl/]

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Remove this file

Reply:
Bit late to the party, my man :) Good spot though. I'll remove that the next time I put a PR in around this area. Thanks!

Reply:
I agree. I could just do a NOJIRA PR i suppose